docs/develop/api-reference/dictionary-collections.md

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 1
+sidebar_label: Dictionary collections
+title: Dictionary collection API reference
+description: Learn how to interact with the API for dictionary collection data types in Momento Serverless Cache.
+slug: /develop/api-reference/collections/dictionary
+---
+
+# Dictionary collection API reference
+
+## Dictionary methods
+This page details the Momento API methods for [dictionary collections](./../datatypes.md#dictionary-collections).
+
+### DictionaryFetch
+
+Gets a dictionary item from a cache.
+
+| Name            | Type   | Description                                   |
+| --------------- | ------ | --------------------------------------------- |
+| cacheName       | String | Name of the cache.                            |
+| dictionaryName  | String | The name of the dictionary to be retreived. |
+
+<details>
+  <summary>Method response object</summary>
+
+The response object for DictionaryFetch returns three possible options, a cache hit, miss, or an error.
+
+* Cache hit
+    - `valueDictionaryBytesBytes()`: Map<Bytes, Bytes>
+    - `valueDictionaryStringString()`: Map<String, String>
+    - `valueDictionaryStringBytes()`: Map<String, Bytes>
+    - `valueDictionaryBytesString()`: Map<Bytes, String>
+    - `toString()`: string - displays the key/value pairs, truncated.
+* Cache miss
+* Cache error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### DictionaryGetField
+Get one field from a dictionary item in the cache.
+
+| Name            | Type         | Description                                   |
+| --------------- | ------------ | --------------------------------------------- |
+| cacheName       | String       | Name of the cache.                            |
+| dictionaryName  | String       | Name of the dictionary to be retreived. |
+| field           | String/bytes | Name of the field in the dictionary to be retreived. |
+
+<details>
+  <summary>Method response object</summary>
+
+* Cache hit
+    - `fieldString()`: string
+    - `fieldBytes()`: bytes
+    - `valueString()`: string
+    - `valueBytes()`: bytes
+
+* Cache miss
+    - `fieldString()`: string
+    - `fieldBytes()`: bytes
+
+* Cache error
+    - `fieldString()`: string
+    - `fieldBytes()`: bytes
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### DictionaryGetFields
+Get one or more fields from a dictionary item in the cache.
```

Comment:
same as elsewhere, basically all places where you have "dictionary item" I would probably see how you feel about the text with just "dictionary" instead.